plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/grouped-product-selector/product-item-label/block.json

```diff
@@ -0,0 +1,47 @@
+{
+	"name": "woocommerce/add-to-cart-with-options-grouped-product-selector-item-label",
+	"title": "Product Item Label",
```

Comment:
Should it have a different name? Product title instead?

Reply:
Or `Product Label`?